### PR TITLE
Lab 4

### PR DESCRIPTION
**Pull request by Jinontheclock**
Lab 4 first pull request

### CODE REVIEW

File: stables.js [hunk 1]
```diff
@@ -43,9 +43,7 @@ console.log(rentIntroduction);
 let horseNickName = "Cookie";
 console.log(`My house nickname is ${horseNickName}`);
```
Comment (kass-lynd) [inline stables.js:44]: Spelling for horse

File: stables.js [hunk 2]
```diff
@@ -43,9 +43,7 @@ console.log(rentIntroduction);
...
 
+
 const rent = 1000
-const food = 1000
-const shopping = 500
 
-let STABLE_MONTHLY = rent + food + shopping;
-console.log(STABLE_MONTHLY/10 * 3);
\ No newline at end of file
+console.log(rent/10 * 3);
```
Comment (kass-lynd) [inline stables.js:49]: formula is right, provide description for savings, add $ sign

File: stables.js (hunk 1 above)
Comment (kass-lynd) [inline stables.js:44]: Create introduction for stables

File: stables.js [hunk 3]
```diff
@@ -40,10 +40,16 @@ let rentIntroductions = `It costs ${STABLE_MONTHLY_FEE} to board ${horeseName}`;
 console.log(rentIntroduction);
 
 //////////////Lesson 4 code//////////////
+const rent = 1000
 let horseNickName = "Cookie";
-console.log(`My house nickname is ${horseNickName}`);
+let introductionHorse = `My horse nickname is ${horseNickName}`;
```
Comment (kass-lynd) [inline stables.js:45]: horse's

File: stables.js [hunk 4]
```diff
@@ -40,10 +40,16 @@ let rentIntroductions = `It costs ${STABLE_MONTHLY_FEE} to board ${horeseName}`;
...
+let introductionStable = `Welcome to ${horseNickName}'s stable.`;
+let discount = `sign up today and receive %10 off for three months!`;
```
Comment (kass-lynd) [inline stables.js:47]: Capital S in sign
Capital T in that

**Comment by Jinontheclock** [comment]
Finished!

**Review by kass-lynd — COMMENTED** [review]
Looks great!

**Pull request merged**